.github/workflows/deploy-published-releases-cdn.yaml

```diff
@@ -28,31 +26,9 @@ jobs:
         if: steps.cache-dependencies.outputs.cache-hit != 'true'
         run: npm ci
 
-      - name: Build package
-        run: npm run build
```

Comment:
It still needs a build before sending to the CDN, doesn't it?

Reply:
The other command already builds as part of the routine.